zanata-client-commands/src/main/java/org/zanata/client/commands/push/RawPushCommand.java

```diff
@@ -241,6 +423,25 @@ public void visit(LocaleMapping locale,
     }
 
     /**
+     * Search and return file type with given source file extension
+     *
+     * @param fileTypes
+     * @param fileExtension
+     * @return
+     */
+    private @Nullable DocumentType getFileType(Map<DocumentType, Set<String>> fileTypes,
```

Comment:
- [ ] Is there a method in `DocumentType` that can do this? Or is `fileTypes` a narrower set of types? Maybe this should go into DocumentType anyway (it would just need a `Set<DocumentType>` and `String` since it already knows which source extensions are valid for each).